### DISCUSSION TITLE
Защита аккаунта

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#162] [Show and tell]
Мой аккаунт был утерян
Прошу или забанить аккаунт, или перенести все привилегии со старого аккаунта на айди 2F665AF97FA6F0EF

Айди старого аккаунта C002224F3666744D

**Comment by kkohno** [comment]
в данном репозитории мы не занимаемся восстановлением аккаунтов
